pom.xml

```diff
@@ -94,6 +107,12 @@
             <version>1.3.0</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.bstats</groupId>
+            <artifactId>bstats-sponge</artifactId>
+            <version>2.2.1</version>
```

Comment:
No, never bstats.

Reply:
Why

Reply:
Because peoples data should be private. The only time I expect to see other server data is when the admin has given the data knowingly (such as sending it over via discord). I personally feel violated when any software attempts to steal my data, especially when they have a option to disable the mertics yet they steal it anyway ... Jus less. 

I will never have metrics in my software.

Reply:
oh lmao ok